src/__tests__/convert-spec.js

```diff
@@ -0,0 +1,62 @@
+import {convertContentTo, convertContentFrom} from '../lib/convert';
+import DraftJS, {ContentState} from 'draft-js';
+
+describe('convertContentTo', () => {
+
+	let draftJsContent;
+	beforeEach(() => {
+		draftJsContent = DraftJS.ContentState.createFromText('<div>some <a href="#">content</a></div>');
```

Comment:
If you want to create ContentState JSON object from a HTML block, you will have to use [convertFromHTML](https://draftjs.org/docs/api-reference-data-conversion.html#convertfromhtml). createFromText will use the HTML tags as a regular string.

Reply:
The actual content doesn't really matter for these tests the way they are now.

Reply:
would it be possible to use a string without HTML tags then?

Reply:
of course.